model/device.go

```diff
@@ -93,6 +93,146 @@ const (
 	DevTypeTest       = "Test"
 )
 
+// Consts containing defaults for all device configs.
+const (
+	defaultActPeriod int64 = 60
+	defaultMonPeriod int64 = 60
+)
+
+// Consts containing defaults for a controller config.
+const (
+	DefaultControllerInputs  string = "A0,X50,X51,X60,X10"
+	DefaultControllerOutputs string = "D14,D15,D16" // Peripheral Power.
+)
+
+// ControllerDefaultVars is a map which relates controller variable names to their default values
+var DefaultControllerVars = map[string]string{
+	"AlarmNetwork":         "10",
+	"AlarmPeriod":          "5",     // 5 seconds.
+	"AlarmRecoveryVoltage": "840",   // 25.4545 V.
+	"AlarmVoltage":         "825",   // 25 V.
+	"AutoRestart":          "600",   // 10 minutes.
+	"Power1":               "false", // initialise to OFF.
+	"Power2":               "false", // initialise to OFF.
+	"Power3":               "false", // initialise to OFF.
+	"Pulses":               "3",     // 3 pulses per cycle.
+	"PulseWidth":           "2",     // 1s ON, 1s OFF.
+	"PulseCycle":           "30",    // 30s between flashes.
+	"PulseSuppress":        "false",
+}
+
+// ControllerDefaultSensors contains a slice of the default sensors for a controller device.
+var DefaultControllerSensors = []SensorV2{
```

Comment:
I almost think each one of these should have a factory function. e.g.

```
func batteryVoltageSensor() SensorV2{
    return SensorV2{Name: "Battery Voltage", Pin: BatteryVoltage, Quantity: dcv, Func: scale, Args: voltageScaleFactor, Units: voltage, Format: round1}
}

// other factories
````

Also notice that I'm making a bunch of stuff consts too (which could from a "string enum") this ensures we have a set of valid values for each field defined. We could have the set of valid Funcs for example, 
```
type Func string

const (
    Scale Func = "scale"
    Linear Func = "linear"
    // etc....
)
```


then have a factory for the default list of sensors,

````
func defaultControllerSensors() []SensorV2 {
    return []SensorV2{
        batteryVoltageSensor(),
        airTemperatureSensor(),
    }
```